modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/parsers/DescribeFeatureTypeResponseFactory.java

```diff
@@ -40,7 +40,7 @@ public boolean isAvailable() {
     public boolean canProcess(WFSRequest originatingRequest, String contentType) {
         return originatingRequest instanceof DescribeFeatureTypeRequest
                 && (contentType == null || contentType.startsWith("text/xml")
-                || contentType.startsWith("application/gml+xml"));
+                || contentType.startsWith("application/gml+xml")|| contentType.startsWith("application/xml"));
```

Comment:
This seems unrelated

Reply:
It is but for some reason the test was failing here.